### PR TITLE
1363 bug incorrect usage of storage objects

### PR DESCRIPTION
**Pull request by Wagnerp**
* #1363 - 'Removed' duplicates

image

### CODE REVIEW

File: Source/Krypton Components/Krypton.Toolkit/Translations/Converters/ToastNotificationIconStrings.cs [hunk 1]
```diff
@@ -23,55 +51,197 @@ public ToastNotificationIconStrings()
...
 
-        [Browsable(false)] 
-        public bool IsDefault => true;
+        #endregion
+
+        #region Public
 
+        /// <summary>Gets or sets the application icon string.</summary>
+        [Localizable(true)]
+        [Category(@"Visuals")]
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Translations/Converters/ToastNotificationIconStrings.cs:87]: Everytime you have added a new designer element, I have had to add a `Reset..` and `ShouldSerialize..` methods to ensure that the default values are not always placed into the designer usage in developers applications.
So please retrofit them to all these new designer elements a well please !

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Components/Krypton.Toolkit/Translations/Converters/ToastNotificationIconStrings.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Translations/Converters/ToastNotificationIconStrings.cs:87]: Ahh... forgot about that...

Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Translations/Converters/ToastNotificationIconStrings.cs:87]: @Smurf-IV Should these be in `ToastNotificationIconStrings` or `KryptonGlobalToolkitStrings`?

Comment (Smurf-IV) [reply Source/Krypton Components/Krypton.Toolkit/Translations/Converters/ToastNotificationIconStrings.cs:87]: Where-ever you have the designer element, then reflection is used on the private API's for the designer to access their states

Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Translations/Converters/ToastNotificationIconStrings.cs:87]: > Where-ever you have the designer element, then reflection is used on the private API's for the designer to access their states

The weird thing is it does have `Reset..` & `ShouldSerialize..` methods in `KryptonGlobalToolkitStrings`?

```cs
/// <summary>Gets the toast notification icon strings.</summary>
        /// <value>The toast notification icon strings.</value>
        [Category(@"Visuals")]
        [Description(@"Collection of toast notification icon strings.")]
        [MergableProperty(false)]
        [DesignerSerializationVisibility(DesignerSerializationVisibility.Content)]
        [Localizable(true)]
        public ToastNotificationIconStrings ToastNotificationIconStrings => ToastNotificationIcon;

        private bool ShouldSerializeToastNotificationIconStrings() => !ToastNotificationIcon.IsDefault;

        /// <summary>Resets the toast notification icon strings.</summary>
        public void ResetToastNotificationIconStrings() => ToastNotificationIcon.Reset();
```

Comment (Smurf-IV) [reply Source/Krypton Components/Krypton.Toolkit/Translations/Converters/ToastNotificationIconStrings.cs:87]: They are "Too high", reflection is done on "Every" designer element.

Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Translations/Converters/ToastNotificationIconStrings.cs:110]: Moved all these to under 'Static Strings'

**Pull request merged**